fix: heading shifted problem fixed

kisharnath opened the pull request.

manniL reviewed (changes requested):
<img width="1327" alt="image" src="https://github.com/manniL/lichter.io/assets/640208/b7656e7f-6bf5-4933-8c18-4023575096dd">
^^^ Before

vvvv With the change
<img width="846" alt="image" src="https://github.com/manniL/lichter.io/assets/640208/cd594fab-d7d6-48ea-b72b-86770f269779">

Removing the class will break the layout

kisharnath commented:
I have updated the code 
I noticed another bug that this same thing happens on 1024px width. I have also fixed that 
here is the screeshot
![Screenshot 2024-04-14 164232](https://github.com/manniL/lichter.io/assets/85046952/fbb2e3ab-c444-4d99-b968-e03458bbbe09)

manniL commented on pages/index.vue:
```diff
@@ -34,11 +34,11 @@ const { data: workshops } = useAsyncData('latest-workshops', () => queryContent(
     <AppSection>
       <div class="md:flex md:flex-row items-center">
-        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1">
+        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1 md:flex-grow-[1.05] lg:flex-grow-[1.05]">
```
```suggestion
        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1 md:flex-grow-[1.05]">
```

This would be enough as it defines it for "medium and bigger" screens

manniL commented on pages/index.vue:
```diff
@@ -34,11 +34,11 @@ const { data: workshops } = useAsyncData('latest-workshops', () => queryContent(
             <div class="flex w-1/2 md:w-full items-center">
               <AppParagraph tag="h1" look="superHeading"
-                class="flex flex-col sm:justify-start font-mono mt-8 md:mt-0 ml-4 md:ml-0">
+                class="flex flex-col sm:justify-start font-mono mt-8 md:mt-0 ml-4 md:ml-0 md:text-[55.1px] lg:text-[83px]">
```
Can we use rem's instead of pixels here? Ideally via predefined classes like `text-xl`? 🤔

kisharnath commented:
ok I will try
